src/overview.rst

```diff
@@ -0,0 +1,328 @@
+.. _overview:
+
+What's New in Cylc 8.0b0?
+=========================
+
+**A quick guide for Cylc 7 users.**
+
+.. tip::
+
+   If you need help using Cylc 8 please post questions to the
+   `Cylc Forum <https://cylc.discourse.group/>`_
+
+
+Cylc 8 differs from Cylc 7 in many ways: architecture, scheduling
+algorithm, security, UIs, and working practices, and more.
+
+Backward Compatibility
+----------------------
+
+To make the transition easier, Cylc 8 can run Cylc 7 workflows out of the box.
+
+.. warning::
+
+   But please take action on any deprecation warnings.
+
+
+Terminology and Config File Name
+--------------------------------
+
+- *suite* is now **workflow** - a more widely understood term
+   - the `suite.rc` config file is now `flow.cylc`
+   - *suite daemon* (or *suite server program*) is now **scheduler**
+- *batch system* is now **job runner**
+   - our job runners are not all "batch systems"
+
+Architecture
+------------
+
+Cylc has been re-architected to support a remote web UI:
+
+- a new Hub component, where you authenticate
+   - can run as a regular or privileged user
```

Comment:
**tldr;** indentation correction

```suggestion
  - can run as a regular or privileged user
```

Explanation: https://matrix.to/#/!hwZqSYihGPuhDdIzIP:matrix.org/$16177080099271219Ujjpz:matrix.org?via=matrix.org